content/contributing/style-guide-and-content-model/style-guide.md

```diff
@@ -68,6 +68,14 @@ For example, [AUTOTITLE](/account-and-profile/setting-up-and-managing-your-githu
 > [!TIP]
 > When you @mention an organization, only those that you're a member of will autocomplete. You can still @mention organizations that you're not a member of, like a previous employer, but the organization name won't autocomplete for you.
 
+#### Important
+
+
+Highlights key information users need to know to achieve their goal.
+
+> [!IMPORTANT]
+> Key information users need to know to achieve their goal.
```

Comment:
Adding a real-life example to keep the new content consistent with what we've done for the other alert types.

```suggestion
> Runner Scale Sets do not support multiple labels, only the name of the runner can be used in place of a label. See [AUTOTITLE](/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/deploying-runner-scale-sets-with-actions-runner-controller).
```